#goal definition
cobots_core/Grip goal
bool wait_for_finish
---
#result definition
bool status
string message
---
#feedback
string message
